src/main/webapp/WEB-INF/tags/fp/recurringDVDetails.tag

```diff
@@ -1,30 +1,48 @@
+<%--
+   - The Kuali Financial System, a comprehensive financial management system for higher education.
+   -
+   - Copyright 2005-2014 The Kuali Foundation
+   -
+   - This program is free software: you can redistribute it and/or modify
+   - it under the terms of the GNU Affero General Public License as
+   - published by the Free Software Foundation, either version 3 of the
+   - License, or (at your option) any later version.
+   -
+   - This program is distributed in the hope that it will be useful,
+   - but WITHOUT ANY WARRANTY; without even the implied warranty of
+   - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   - GNU Affero General Public License for more details.
+   -
+   - You should have received a copy of the GNU Affero General Public License
+   - along with this program.  If not, see <http://www.gnu.org/licenses/>.
+--%>
 <%@ include file="/jsp/sys/kfsTldHeader.jsp"%>
 
 <kul:tab tabTitle="Recurring Details (${KualiForm.recurringDVDetailsSize })" defaultOpen="${KualiForm.recurringDVDetailsDefaultOpen}">
 	<c:set var="recurringDVAttributes" value="${DataDictionary.RecurringDisbursementVoucherDetail.attributes}" />
-	<div class="tab-container" align=center > 
-		<table cellpadding=0 class="datatable" summary="Recurring DV Details">
-			<tr>
+	<div class="tab-container"> 
+		<table class="standard side-margins" summary="Recurring DV Details">
+			<tr class="header">
 				<kul:htmlAttributeHeaderCell attributeEntry="${recurringDVAttributes.dvCheckDate}" hideRequiredAsterisk="true" scope="col"/>
 				<kul:htmlAttributeHeaderCell attributeEntry="${recurringDVAttributes.dvCheckAmount}" hideRequiredAsterisk="true" scope="col"/>
 				<kul:htmlAttributeHeaderCell attributeEntry="${recurringDVAttributes.dvCheckStub}" scope="col"/>
 				<kul:htmlAttributeHeaderCell attributeEntry="${recurringDVAttributes.dvDocumentNumber}" hideRequiredAsterisk="true" scope="col"/>
 			</tr>
 			<logic:iterate indexId="ctr" name="KualiForm" property="document.recurringDisbursementVoucherDetails" id="currentDetail">
 				<tr>
-					<td>
+					<td class="datacell">
```

Comment:
You might want to consider adding this class to the <tr> inside the <logic:iterate> tag to add the every other row shading effect:

class="${ctr % 2 == 0 ? "highlight" : ""}"